### PR TITLE
Add SwitchFrameParent()

### LINKED ISSUE #12: No way to switch to frame parent
I've done `SwitchFrame()` to switch into an iframe and perform some actions, but then I need to switch out of it again and do something from outside that frame.

The wisdom on the internet seems to be that I either need to pass an integer "0" to switch to the frame with the first index, or call [`/session/{id}/frame/parent`](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidframe), neither of which seem possible in `go-selenium`. Would a pull request be accepted which adds `SwitchFrameParent()`?

### PR DESCRIPTION
**Pull request by pwaller**
Unfortunately, I whipped up this code, and then discovered that I didn't need it in my use case. So it hasn't been exercised! But I didn't want to throw away the work in case someone else finds it useful (I feel like this will inevitably be me before long…)

Resolves #12

### CODE REVIEW

File: remote.go [hunk 1]
```diff
@@ -466,6 +466,10 @@ func (wd *remoteWebDriver) SwitchFrame(frame string) error {
 	return wd.voidCommand("/session/%s/frame", params)
 }
 
+func (wd *remoteWebDriver) SwitchFrameParent() error {
+	return wd.voidCommand("/session/%s/frame/parent", nil)
```
Comment (sqs) [inline remote.go]: I am on a plane and have forgotten much of the context of this code, so excuse me if this is obvious—but is this the correct URL? Do you need to be able to pass in a frame index? Won't the `%s` expand to nil?

Comment (pwaller) [inline remote.go]: My interpretation of the code is that `%s` is the session ID. [`/frame/parent`](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidframe) doesn't take a frame as a parameter, it just goes to the parent of the current frame.

Comment (sqs) [inline remote.go]: Right. I see that in the other methods now.

**Comment by sqs** [comment]
This LGTM from a quick inspection. Thanks!

**Pull request closed without merging**